Containers/mastercontainer/mastercontainer.conf

```diff
@@ -1,6 +1,10 @@
 Listen 8000
 Listen 8080
 
+# Include ssl modules
+LoadModule ssl_module modules/mod_ssl.so
+LoadModule socache_shmcb_module modules/mod_socache_shmcb.so
```

Comment:
For what is this module (socache_shmcb) needed? And why do we include this here and not in the main httpd.conf?

Reply:
I could add it to the main httpd.conf, it was former in the ssl.conf that I removed. Will add it to the main httpd.conf

Reply:
Would be better that all modules get loaded in one conf

Reply:
jup,, done

Reply:
> I could add it to the main httpd.conf, it was former in the ssl.conf that I removed. Will add it to the main httpd.conf

It seems that the socache_shmcb module is required for the ssl module

Reply:
let me add this then back

Reply:
done. pleae have another look